docs/guides/getting-started/prerequisites.md

```diff
@@ -3,22 +3,204 @@ sidebar_position: 1
 pagination_next: guides/getting-started/beginning-tutorial
 ---
 
-import OSList from '@theme/OSList'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # Prerequisites
 
-- Rust
-- Node (optional)
-- OS specific build tooling, eg. `build-essential`, `xcode-select` or `C++ build tools`
+The first step is to install [Rust] and system dependencies. Keep in mind that this setup is only needed for _developing Tauri apps_. Your end-users are not required to do any of this.
 
-For more information on platform-specific build tooling, see these guides to get started.
+## Setting Up Windows
 
-<OSList content={{
-    linux: { title: 'Linux Setup', link: './setup/linux'},
-    macos: { title: 'macOS Setup', link: './setup/macos'},
-    windows: { title: 'Windows Setup', link: './setup/windows'}
-}} />
+:::info
+For those using the Windows Subsystem for Linux (WSL), please refer to our [Linux specific instructions](#setting-up-linux) instead.
+:::
 
-After that, you'll be ready to [make your first Tauri app][beginning tutorial]!
+On Windows, go to [https://www.rust-lang.org/tools/install][install rust] to install `rustup` (the Rust installer). You also need to install Microsoft Visual Studio C++ build tools. The easiest way is to install [Build Tools for Visual Studio 2019]. When asked which workloads to install, ensure "C++ build tools" and the Windows 10 SDK are selected.
 
-[beginning tutorial]: beginning-tutorial.md
+### Install WebView2
+
+:::note
+WebView2 is pre-installed in Windows 11
+:::
+
+Tauri heavily depends on WebView2 to render web content on Windows, therefore you must have WebView2 installed. The easiest way is to download and run the Evergreen Bootstrapper from the [official website][download webview2].
+
+The bootstrapper script will try to determine the correct architecture and version for your system. Still, if you run into issues - especially with Windows on ARM - you can select the correct standalone installer or even a fixed version.
+
+## Setting Up macOS
+
+To install Rust on macOS, open a terminal and enter the following
+command:
+
+```console
+curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
```

Comment:
@JonasKruckenberg is it possible to just use the command `curl https://sh.rustup.rs | sh`? Seems better for succinctness

Reply:
it's the "official" rustup command (https://rustup.rs/). i don't know the reasons, but i can't imagine that they doing this for fun 🤔 

We have a similar problem on windows with the webview2 installation on windows7 because it doesn't use tls1.2 by default there, but getting the webview2 installer requires that. Maybe it's something similar here

Reply:
Okay, that makes sense to keep as-is